.ci_scripts/runIntegration.sh

```diff
@@ -14,10 +14,16 @@ else
     SBT_ARGS="$SBT_ARGS -Dtest.authMode=cr"
 fi
 
-if [ "$MONGO_PROFILE" = "invalid-ssl" -o "$MONGO_PROFILE" = "mutual-ssl" ]; then
+if [ "$MONGO_PROFILE" != "x509" ]; then
+    TEST_OPTS="$TEST_OPTS,x509"
+else
+    TEST_CLASSES=X509DriverSpec
```

Comment:
In case of X509, not only that tests must be executed, but all the integration tests to cover there is no regression/side-effect due to that auth.